README.md

```diff
@@ -1,6 +1,7 @@
 # COSMIC Desktop
 
-Currently an incomplete **pre-alpha**.
+Currently an incomplete **pre-alpha**. Standard Installation is available for Pop!_OS, Arch Linux and its decendents, like Manjaro.
```

Comment:
I don't mind having specific instructions for some distributions, but in general the systemd-sysext method works on a lot of distributions for testing. Which is all this should be used for right now. So please drop or adjust this line.

```suggestion
Currently an incomplete **pre-alpha**. Testing instructions below for various distributions.
```